src/app/public/src/modules/nav/nav.component.ts

```diff
@@ -41,6 +41,11 @@ export class StacheNavComponent implements OnInit, StacheNav {
   public navigate(route: any): void {
     let extras: any = {};
 
+    if (route.isExternalLink) {
```

Comment:
Here's another thought. What if one of the routes contains http for some reason? Like an http topic?